diff -Naurp a/config.sub b/config.sub
--- a/config.sub	2017-01-17 01:57:54.000000000 -0600
+++ b/config.sub	2017-01-17 11:19:07.418837420 -0600
@@ -1040,7 +1040,7 @@ case $basic_machine in
 		;;
 	ppc64)	basic_machine=powerpc64-unknown
 		;;
-	ppc64-*) basic_machine=powerpc64-`echo $basic_machine | sed 's/^[^-]*-//'`
+	ppc64-* | ppc64p7-*) basic_machine=powerpc64-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
 	ppc64le | powerpc64little)
 		basic_machine=powerpc64le-unknown
